MAUI/Scheduler/agenda-view.md

```diff
@@ -0,0 +1,301 @@
+{
+    Subject = "Meeting",
+    StartTime = DateTime.Now,
+    EndTime = DateTime.Now.AddHours(1),
+    Background = Brush.Orange,
+});
+// Adding scheduler appointment into the AppointmentsSource.
+this.Scheduler.AppointmentsSource = appointments;
+
+{% endhighlight %}
+{% endtabs %}
+
+## Month header customization
+
+The agenda month header view can be customized by using the `MonthHeaderSettings` property of `AgendaView` in the `SfScheduler.`
+
+### Customize month header appearance using style
+
+You can style the date format, height, text style, and background color by using the properties such as `DateFormat,` `Height,` `TextStyle,` and `Background` properties of `MonthHeaderSettings.`
+
+{% tabs %}
+{% highlight xaml %}
+
+ <scheduler:SfScheduler x:Name="Scheduler"
+                        View="Agenda">
+    <scheduler:SfScheduler.AgendaView>
+        <scheduler:SchedulerAgendaView>
+            <scheduler:SchedulerAgendaView.MonthHeaderSettings>
+                <scheduler:SchedulerMonthHeaderSettings DateFormat="MMM yyy"
+                                                        Height="200"
+                                                        Background="LightGreen" />
+            </scheduler:SchedulerAgendaView.MonthHeaderSettings>
+        </scheduler:SchedulerAgendaView>
+    </scheduler:SfScheduler.AgendaView>
+ </scheduler:SfScheduler>
+
+{% endhighlight %}
+{% highlight c# %}
+
+this.Scheduler.View = SchedulerView.Agenda;
+var textStyle = new SchedulerTextStyle()
+{
+    TextColor = Colors.Red,
+    FontSize = 25,
+};
+
+this.Scheduler.AgendaView.MonthHeaderSettings.DateFormat = "MMM yyy";
+this.Scheduler.AgendaView.MonthHeaderSettings.Height = 200;
+this.Scheduler.AgendaView.MonthHeaderSettings.TextStyle = textStyle;
+this.Scheduler.AgendaView.MonthHeaderSettings.Background = Brush.LightGreen;
+
+{% endhighlight %}
+{% endtabs %}
+
+### Customize month header appearance using DataTemplate
+
+You can customize the month header appearance of scheduler by using the `MonthHeaderTemplate` property of `AgendaView.`
+
+{% tabs %}
+{% highlight xaml %}
+
+  <scheduler:SfScheduler x:Name="Scheduler"
+                         View="Agenda">
+    <scheduler:SfScheduler.AgendaView>
+        <scheduler:SchedulerAgendaView>
+            <scheduler:SchedulerAgendaView.MonthHeaderTemplate>
+                <DataTemplate>
+                    <Grid>
+                        <Label x:Name="label" HorizontalOptions="Center" Background="LightGreen" VerticalOptions="Center" TextColor="Black" FontSize="25"  Text="{Binding StringFormat='{0:MMMM yyyy}'}" />
+                    </Grid>
+                </DataTemplate>
+            </scheduler:SchedulerAgendaView.MonthHeaderTemplate>
+        </scheduler:SchedulerAgendaView>
+    </scheduler:SfScheduler.AgendaView>
+ </scheduler:SfScheduler>
+
+{% endhighlight %}
+{% endtabs %}
+
+N> 
+* The default value of `DateFormat,` and `Height` are `MMMM yyyy,` and `150` respectively.
```

Comment:
The default values for `DateFormat` and `Height` are `MMMM yyyy` and `150`, respectively.